src/components/subscriptions/SubscriptionManagementPage.jsx

```diff
@@ -1,115 +1,47 @@
-                    {({
-                      details,
-                      fetchSubscriptionUsers,
-                      fetchSubscriptionDetails,
-                      setActiveTab,
-                      activeTab,
-                    }) => (
-                      <>
-                        <p className="lead">
-                          {details.licenses.allocated}
-                          {' of '}
-                          {details.licenses.total} licenses allocated
-                        </p>
-                        <div className="my-3 row">
-                          <div className="col-12 col-md-5 mb-3 mb-md-0">
-                            <SearchBar
-                              placeholder="Search by email..."
-                              onSearch={searchQuery => fetchSubscriptionUsers({ searchQuery })}
-                              onClear={() => fetchSubscriptionUsers()}
-                            />
-                          </div>
-                          {(details.licenses.allocated > 0 || activeTab !== TAB_ALL_USERS) && (
-                          <div className="col-12 col-md-7 text-md-right">
-                            <AddUsersButton
-                              onSuccess={({ numAlreadyAssociated, numSuccessfulAssignments }) => {
-                                fetchSubscriptionDetails();
-                                addToast(`${numAlreadyAssociated} email addresses were previously assigned. ${numSuccessfulAssignments} email addresses were successfully added.`);
-                                setActiveTab(TAB_PENDING_USERS);
-                              }}
-                            />
-                          </div>
-                          )}
-                        </div>
-                      </>
-                    )}
-                  </SubscriptionConsumer>
-                </div>
-                <div className="row my-4">
-                  <div className="col-12 col-lg-3 mb-2 mb-lg-0">
-                    <LicenseAllocationNavigation />
-                  </div>
-                  <div className="col-12 col-lg-9">
-                    <TabContentTable />
-                  </div>
-                </div>
-              </div>
-            </div>
-          </div>
-        </main>
-      </SubscriptionData>
-    </>
+      <main role="main" className="manage-subscription">
+        <Container className="py-3" fluid>
+          <Switch>
+            <Route
+              path="/:enterpriseSlug/admin/subscriptions"
+              component={MultipleSubscriptionsPage}
+              exact
+            />
+            <Route
+              path="/:enterpriseSlug/admin/subscriptions/:subscriptionUUID"
+              component={SubscriptionDetailPage}
+              exact
+            />
+          </Switch>
+        </Container>
+      </main>
+    </SubscriptionData>
   );
 }
 
 SubscriptionManagementPage.propTypes = {
-  enterpriseSlug: PropTypes.string.isRequired,
```

Comment:
I like this! was not sure earlier when I looked at this code why both were being used here